tests/bluetooth/controller/common/src/helper_util.c

```diff
@@ -321,7 +322,7 @@ void lt_tx_real(const char *file, uint32_t line, helper_pdu_opcode_t opcode, str
 	sys_slist_append(&lt_tx_q, (sys_snode_t *)rx);
 }
 
-void lt_rx_real(const char *file, uint32_t line, helper_pdu_opcode_t opcode, struct ll_conn *conn,
+void lt_rx_real(const char *file, uint32_t line, enum helper_pdu_opcode opcode, struct ll_conn *conn,
```

Comment:
line is now over 100 chars
```suggestion
void lt_rx_real(const char *file, uint32_t line, enum helper_pdu_opcode opcode, 
                       struct ll_conn *conn,
```